### ISSUE TITLE
Rotor landmarks repeat in voiceover

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly-react#3413]
**Describe the issue. What is the expected and unexpected behavior?**
This is likely due to having two components with `role=main` (our page and the examples). Related issue: https://github.com/patternfly/patternfly-next/issues/2079

**Please provide the steps to reproduce. Feel free to link CodeSandbox or another tool.**

**Is this a bug or enhancement? If this issue is a bug, is this issue blocking you or is there a work-around?**
I would consider this more of an enhancement. It could be argued that the difference could be inferred, but it seems unclear to me and should be improved.

**What is your product and what release version are you targeting?**

**Comment by jessiehuff** [comment]
Our documentation has improved a lot over time, and the only remaining components that have significantly repeated landmarks now make sense (such as application launcher and pagination). I imagine that products won't have a ton of each of these components on a page so this is probably fine.
